### ISSUE TITLE
can‘t use:pip install jupyterlab-drawio

### ISSUE THREAD

**Issue by HaresHu** [QuantStack/jupyterlab-drawio#72]
ERROR: Could not find a version that satisfies the requirement jupyterlab-drawio
ERROR: No matching distribution found for jupyterlab-drawio

**Comment by squeezer44** [comment]
Same for me.
Addtionally it seems the module is not known at https://pypi.org/search/?q=jupyterlab-drawio.


- Expected result
`pip install jupyterlab-drawio` installs the module (as mentioned in the docu: https://github.com/QuantStack/jupyterlab-drawio#install) and get it ready to be used inside a running jupyterlab environment.

- Given result
Shortly after command `pip install jupyterlab-drawio` was executed this error message popped up:

```
ERROR: Could not find a version that satisfies the requirement jupyterlab-drawio
ERROR: No matching distribution found for jupyterlab-drawio
```

- System environment
OS: Windows 10
Python 3.8.2
jupyterlab                        3.0.5
jupyterlab-git                    0.23.3
jupyterlab-pygments               0.1.2
jupyterlab-server                 2.1.2
jupyterlab-widgets                1.0.0

**Comment by SylvainCorlay** [comment]
The update to jupyterlab 3.0, which should enable the installation from pip / mamba / conda was just merged but a release has not been made yet.

**Comment by squeezer44** [comment]
@SylvainCorlay 
Does that mean in practice the status stays unchanged until next release of jupyterlab-drawio will be published?

**Comment by wolfv** [comment]
just released 0.8.0 to PyPI that should fix this.

**Comment by squeezer44** [comment]
Regarding : https://github.com/QuantStack/jupyterlab-drawio/issues/72#issuecomment-761881755
Checked and confirmed: `pip install jupyterlab-drawio` is working as expected.

Thx :-)
